index.js

```
@@ -0,0 +1,20 @@
+
+
+function daysInMonth(month, year) {
```

the function should throw an error, do not catch an exception inside